spec/support/domain/shared_context.rb

```diff
@@ -13,11 +13,16 @@
   def dispatch(command)
     gateway.dispatch(command)
   end
+  alias_method :when_command, :dispatch
 
   def given_events(*events)
     stream_id = events.first.id
     events = events.map { |event| journal.push(event, {}) }
-    event_store.append("GameAggregate", stream_id, events)
+    event_store.given_events('GameAggregate', stream_id, events)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.